web-app/js/portal/filter/BooleanFilter.js

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2015 IMOS
+ *
+ * The AODN/IMOS Portal is distributed under the terms of the GNU General Public License
+ *
+ */
+
+Ext.namespace('Portal.filter');
+
+Portal.filter.BooleanFilter = Ext.extend(Portal.filter.Filter, {
+
+    getSupportedGeoserverTypes: function() {
+
+        return ['boolean'];
```

Comment:
just wondering, should these types be configised somewhere, to avoid them being hardcoded into each filter type?